packages/ignite-basic-structure/index.js

```diff
@@ -54,6 +54,9 @@ const add = async function (context) {
     filesystem.copy(`${sourceFolder}/App`, `${process.cwd()}/App`, { overwrite: true })
   }
 
+  // Create ./ignite/plugins
+  filesystem.dir(`${process.cwd()}/ignite/plugins`)
```

Comment:
Woo!

Maybe replace with this?

```js
filesystem.write(`${process.cwd()}/ignite/plugins/.gitkeep`, '')
```

Currently reading plugins from this directory isn't enabled yet.  We'd have to add another line like this to tell gluegun to load all plugins found in that directory.

https://github.com/infinitered/ignite/blob/next/packages/ignite/src/cli/cli.js#L18

This time it would be `hidden: false` though.  We still have some work to do convincing @GantMan that this is a good idea, but gluegun supports this out of the box.  🎉